joomla/joomla-cms
# [4.0] Remove caption js

Pull Request to remove the legacy caption.js

### Summary of Changes
Removes the JHtml::_('behavior.caption') calls, the method itself and its JS files and test code.


### Testing Instructions
* Make sure caption.js isn't requested anymore in any pageview
* Captions should work as before because we actually moved to use `figure` and `figcaption` long ago. The JS was kept and loaded for B/C reasons with existing old articles.


### Expected result
Captions work as before


### Actual result
Captions work as before


### Documentation Changes Required
Mark the method as deprecated in 3.9 (I can do that PR) and in the docs.
